src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/MoveDownCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Packaging;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    [ProjectCommand(ManagedProjectSystemPackage.ManagedProjectSystemOrderCommandSet, ManagedProjectSystemPackage.MoveDownCmdId)]
+    [AppliesTo(ProjectCapability.FSharp)]
```

Comment:
Could these (and the vsct file, etc), just be in the FSharp dll?  Any reason for having them here?

Reply:
I haven't tried moving those into the Managed.FSharp.VS project. But, in Managed.VS we have the CSharp and VB specific commands together, ex: AddClassProjectCSharpCommand.cs, AddClassProjectFSharpCommand.cs.

From that I assumed it was ok to have the commands in Managed.VS.

Reply:
It's not the *worst*, it just means that dll is bigger, and takes up address space for all projects, even though they will never be applicable, etc.

It also can make it harder to find things.

I haven't been around project-system long enough to know if there is some reason this needs to be like this though.  Maybe someone else will tell me this is right.

Reply:
I agree it should be in FSharp.VS, perhaps I can try to see what happens :)

Reply:
Yea, maybe we should discuss this part more.